packages/calcite-components/src/components/tab-nav/tab-nav.tsx

```diff
@@ -306,7 +349,18 @@ export class TabNav {
 
   animationActiveDuration = 0.3;
 
-  resizeObserver = createObserver("resize", () => {
+  @State() defaultMessages: TabNavMessages;
+
+  @State() effectiveLocale = "";
+
+  @Watch("effectiveLocale")
+  effectiveLocaleChange(): void {
+    updateMessages(this, this.effectiveLocale);
+  }
+
+  @State() elWidth: number;
```

Comment:
What is this state used for? Do we need it?

Reply:
Nope. Relic of the past.